### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by maheshlalam**
favorite books and movies list

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,11 +6,25 @@
     <title>studentsite</title>
 </head>
 <body>
-    hi i am mahesh
+    hi
+    i am mahesh
```
Comment (friendly-developer) [inline index.html:10]: use "I" instead of small i

File: index.html [hunk 2]
```diff
@@ -6,11 +6,25 @@
...
+    1)To Kill a Mockingbird, by Harper Lee. ...
+    2)1984, by George Orwell. ...
+    3)Harry Potter and the Philosopher's Stone, by J.K. Rowling.
+    4)The Lord of the Rings, by J.R.R. ...
+    5)The Great Gatsby, by F. Scott Fitzgerald. ...
+    
+
+    My favorite movies are
+    1)bahubali
+    2)Jhon wick
```
Comment (friendly-developer) [inline index.html:25]: John*

File: index.html [hunk 3]
```diff
@@ -6,11 +6,25 @@
     <title>studentsite</title>
 </head>
 <body>
-    hi i am mahesh
+    hi
+    i am mahesh
     i worked at STBL,
     And completed =-graduation at Gitam university visakhapatnam
```
Comment (friendly-developer) [inline index.html]: Fix typos

**Review by friendly-developer — CHANGES_REQUESTED** [review]

**Review by friendly-developer — APPROVED** [review]
Looks good! Go ahead and merge.

**Comment by maheshlalam** [comment]
fixed

**Pull request merged**